components/Tabs/Tabs.js

```diff
@@ -59,4 +60,40 @@ class TabItem {
 
 */
 
-links = document.querySelectorAll();
\ No newline at end of file
+const links = document.querySelectorAll('.tabs-link[data-tab]');
+links.forEach((link) => {
+  const tabLink = new TabLink(link)
+});
+
+
+
+
+// slideshow
```

Comment:
great work on the stretch! You should always try to convert your Var to let or const receptively.